packages/default-theme/src/pages/checkout.vue

```diff
@@ -208,6 +214,7 @@ export default {
         // perform a redirection to the external payment gateway
         window.location.href = redirectUrl
       } catch (error) {
+        console.warn("err place order", error)
```

Comment:
`const {devtools} = getApplicationContext()`
```suggestion
        devtools?.warning("[checkout] Error while placing an order", error)
```

Reply:
done